Wordle.Api/Wordle.Api/Services/WordDateService.cs

```diff
@@ -0,0 +1,51 @@
+using Azure.Core;
+using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Dtos;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Services;
+
+public class WordDateService(WordleDbContext Db)
+{
+
+    public async Task<WordDate?> GetWordDate(string date)
+    {
+        return await Db.WordDates.FirstOrDefaultAsync(p => p.Date == date);
+    }
```

Comment:
```suggestion
    public async Task<WordDate?> GetWordDate(DateTime date)
    {
        return await Db.WordDates.FirstOrDefaultAsync(p => p.Date == date);
    }
```

You should use proper format for Date instead of string value

Reply:
I decided to use a string for the date throughout the program because between ts, c#, and .vue, they have some differences in their date formats, so I converted them all to just M/D/Y before passing them along